CHANGELOG.md

```diff
@@ -1 +1,160 @@
 # Changelog
+
+For more information on the list of deprecated flags and APIs, have a look at
+https://docs.docker.com/engine/deprecated/ where you can find the target removal dates 
+
+## 18.06.0-ce (2018-07-DD)
+
+### Important notes about this release
+
+- TODO: Notice here for release cycle change
+
+### Builder
+
+* Builder: fix layer leak on multi-stage wildcard copy. [moby/moby#37178](https://github.com/moby/moby/pull/37178)
+* Fix parsing of invalid environment variable substitution . [moby/moby#37134](https://github.com/moby/moby/pull/37134)
+* Builder: use the arch info from base image. [moby/moby#36816](https://github.com/moby/moby/pull/36816) [docker/cli#1111](https://github.com/docker/cli/pull/1111) [moby/moby#37197](https://github.com/moby/moby/pull/37197)
```

Comment:
Something wrong in here. Cli links is for "experimental buildkit" line. 37197 is duplicated.